util/log/log.go

```diff
@@ -27,6 +27,8 @@ import (
 	"golang.org/x/net/context"
 )
 
+const timeFormat = "1/2 15:04:05.000" // m/d h:m:s.nanos
```

Comment:
I agree with Marc.  The year should be in there.  And we should zero-pad everything. 
yyyy-mm-dd hh:mm-ss.nnn 
The year makes it very clear the order of months and days.